src/consensus/params.h

```diff
@@ -62,6 +62,7 @@ struct Params {
     int64_t nPowTargetTimespan;
     int64_t DifficultyAdjustmentInterval() const { return nPowTargetTimespan / nPowTargetSpacing; }
     uint256 nMinimumChainWork;
+    uint256 defaultAssumeValid;
```

Comment:
This can be in CChainParams instead of Consensus::Params, which is the consensus "section of chainparams".